msg/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ set(msg_files
 	gps_dump.msg
 	gps_inject_data.msg
 	home_position.msg
+	hover_thrust_estimator.msg
```

Comment:
It would be better if this is `hover_thrust_estimate`

Reply:
makes sense